ChangeLog

```diff
@@ -1,7 +1,89 @@
 CHANGES
 =======
 
-* Update to latest model
```

Comment:
Did you mean to commit changes to ChangeLog?

Reply:
Yes, I did Chris. Since pbr doesn't work with poetry just yet, I thought I would manually update the ChangeLog with content from the latest releases. @sierra-moxon is preparing some slides to help integrate pbr with poetry, which will be part of an upcoming PR soon.